functions/venue.js

```diff
@@ -711,6 +711,35 @@ exports.deleteVenue = functions.https.onCall(async (data, context) => {
   admin.firestore().collection("venues").doc(venueId).delete();
 });
 
+exports.deletePollInVenue = functions.https.onCall(
+  async ({ venueId, pollId }, context) => {
+    checkAuth(context);
+    await checkUserIsAdminOrOwner(venueId, context.auth.token.user_id);
+
+    admin
+      .firestore()
+      .collection("venues")
+      .doc(venueId)
+      .collection("chats")
+      .doc(pollId)
+      .update({ deleted: true });
+  }
+);
```

Comment:
No, basically you can use the `deleteVenueMessage`, you don't need a separate function for this. It's that you should only display the UI button, if there a user is an admin & a venue owner

Reply:
got it. Fixed in 55c674d712fa3c88df459d4ed1c2a6b24482e632